frontend/script/index.js

```diff
@@ -0,0 +1,6 @@
+    function iniciarJogo() {
+    //sair da paginia inicial para a de login por esse botão
+    //se nao funcionar por esse caminho avisa
+    window.location.href = "/Battleship/frontend/battleship/login/login.html"; 
```

Comment:
```suggestion
    window.location.href = "../../frontend/battleship/login/login.html"; 
```

O meu funcionou apenas quando troquei o Battleship p/ `../../` verifica aí. Validando isso pode mergear.